backend/api/models.py

```diff
@@ -1,56 +1,99 @@
+# -*- coding: utf-8 -*-
 from django.contrib.gis.db import models
 from django.conf import settings
 
-# TODO: Design and implement these models
-
 class Factory(models.Model):
     """Factories that are potential to be illegal.
+            Containing:
+           - id :  Safety concern (primary key)
+           - fact_name
+           - lat
+           - lng
+           - landcode
+           - fact_type
+           - status :  The status of the processes
+           - status_time :  The last time of updating the status
+           - others :  Attachment of the description
+           - creat_at
+       """
+
+    # List of fact_type & status
+    fact_type_list = [
+        ('1','金屬'),
+        ('2-1','沖床、銑床、車床、鏜孔'),
+        ('2-2', '焊接、鑄造、熱處理'),
+        ('2-3', '金屬表面處理、噴漆'),
+        ('3', '塑膠加工、射出'),
+        ('4', '橡膠加工'),
+        ('5', '非金屬礦物（石材）'),
+        ('6', '食品'),
+        ('7', '皮革'),
+        ('8', '紡織'),
+        ('9', '其他')
+    ]
+    status_list = [
+        ('D','已舉報'),
+        ('F','資料不齊'),
+        ('A','待審核')
+    ]
+
+    # All  Features
+    id =  models.UUIDField(max_length=36, primary_key=True, serialize=False, verbose_name='ID')
```

Comment:
```suggestion
    import uuid
    id =  models.UUIDField(
         primary_key=True, 
         default=uuid.uuid4, 
         editable=False,
         verbose_name='ID',
     )
```
Not sure why we need to disable serializability. Also, we don't need to limit the max length of it, `uuid4` is already a 128 bit string.